robosuite/models/grippers/two_finger_gripper.py

```diff
@@ -23,7 +23,7 @@ def init_qpos(self):
 
     @property
     def joints(self):
-        return ["r_gripper_l_finger_joint", "r_gripper_r_finger_joint"]
+        return ["r_gripper_r_finger_joint", "r_gripper_l_finger_joint"]
```

Comment:
do you remember why this ordering was reversed? just curious

Reply:
for some reason the original mapping was contrary to what the gripper docstring dictated. This fix makes it so that the open / close convention as defined by the gripper docstring is consistent across all grippers (action: 1 => open, -1 => closed)